submissions/gurpreet-mike/scripts/article.js

```diff
@@ -23,7 +23,7 @@
   // DONE: Set up a DB table for articles.
   Article.createTable = function(callback) {
     webDB.execute(
-      'CREATE TABLE articles(title TEXT, category TEXT, author TEXT, authorUrl TEXT, publishedOn TEXT, body TEXT );', // what SQL command do we run here inside these quotes?
+      'CREATE TABLE articles(id INTEGER PRIMARY KEY, title TEXT, category TEXT, author TEXT, authorUrl TEXT, publishedOn TEXT, body TEXT );', // what SQL command do we run here inside these quotes?
```

Comment:
You get the ID for free in webSQL, so you don't need to include it specifically here.